yt_dlp/extractor/vk.py

```diff
@@ -30,8 +30,10 @@ class VKBaseIE(InfoExtractor):
     _WAF_URI = 'https://vk.com/429.html?'
 
     def _download_webpage_handle(self, *args, **kwargs):
-        content, urlh = super(VKBaseIE, self)._download_webpage_handle(
-            *args, **kwargs)
+        try:
+            content, urlh = super()._download_webpage_handle(*args, **kwargs)
+        except TypeError:
+            return False
```

Comment:
too broad! Handle False explictly
```suggestion
        res = super()._download_webpage_handle(*args, **kwargs)
        if not res:
            return res
        content, urlh = res
```